src/bin/mrtg

```diff
@@ -1040,25 +1040,36 @@ sub writegraphics {
 
           my $graphitensprefix="m2g";
 
-          # send the 'in' var data to graphite, and log it appropriately
-          debug('log', "graphite->send($graphitensprefix.$graphiterouter.$graphitein,$inlast,$time)\n");
-          $graphiteObj->send(
-            path => "$graphitensprefix.$graphiterouter.$graphitein",
-            value => "$inlast",
-            time => $time,
-          );
-
-          # send the 'out' var data to graphite, and log it appropriately
-          debug('log', "graphite->send($graphitensprefix.$graphiterouter.$graphiteout,$outlast,$time)\n");
-          $graphiteObj->send(
-            path => "$graphitensprefix.$graphiterouter.$graphiteout",
-            value => "$outlast",
-            time => $time,
-          );
+          # verify that the 'in' value is numeric, and send to graphite
+          if($inlast =~ /\d+/) {
```

Comment:
maybe `$inlast =~ /^\d+$/` while you are at it ? or even a regexp that allows floats ?